### PR TITLE
Add OSS integration documentation

### PR DESCRIPTION
**Pull request by JAORMX**
# Summary

Fills in the docs

## Change Type

***Mark the type of change your PR introduces:***

- [ ] Bug fix (resolves an issue without affecting existing features)
- [ ] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [x] Documentation (updates or additions to documentation)
- [ ] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

***Outline how the changes were tested, including steps to reproduce and any relevant configurations. 
Attach screenshots if helpful.***

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [ ] Added comments to complex or tricky code sections.
- [x] Updated any affected documentation.
- [ ] Included tests that validate the fix or feature.
- [ ] Checked that related changes are merged.

### CODE REVIEW

File: docs/docs/integrations/oss_integrations.md [hunk 1]
```diff
@@ -1,3 +1,201 @@
...
+|             |     exit-code: 1                                                                      |
+|             | ```                                                                                   |
+|             |                                                                                       |
+|             | For more information, see                                                             |
+|             | https://github.com/marketplace/actions/aqua-security-trivy                            |
+|             |                                                                                       |
++-------------+---------------------------------------------------------------------------------------+
+```
+
+If the rule type is enabled and automated remediation is configured, Minder will automatically create a pull request to enable Trivy scanning on your repository.
```
Comment (eleftherias) [inline docs/docs/integrations/oss_integrations.md:66]: nit: we use "automatic remediation" in the rest of the docs

File: docs/docs/integrations/oss_integrations.md [hunk 2]
```diff
@@ -1,3 +1,201 @@
...
+|             | to ensure that vulnerabilities are not introduced into the codebase.                                                             |
+|             |                                                                                                                                  |
+|             | For more information, see                                                                                                        |
+|             | https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file |
+|             |                                                                                                                                  |
++-------------+----------------------------------------------------------------------------------------------------------------------------------+
+```
+
+If the rule type is enabled and automated remediation is configured, Minder will automatically create a pull request to enable Dependabot on your repository.
```
Comment (eleftherias) [inline docs/docs/integrations/oss_integrations.md:113]: automated -> automatic

**Review by eleftherias — CHANGES_REQUESTED** [review]
One comment on consistent terminology:

**Comment by JAORMX** [comment]
@eleftherias resolved. Thanks for the feedback!

**Review by eleftherias — APPROVED** [review]

**Pull request merged**